README.md

```diff
@@ -75,5 +75,13 @@ Current working directory for logging file paths.
 
 Logger function, called once at the end and once per file if `logFiles` is enabled.
 
+##### `options.includeZero: boolean = false`
+
+Set to true if you want to include results with no rows.
+
+##### `options.onlyZero: boolean = false`
```

Comment:
i don't see a lot of value in this feature. why do you want to log only on an empty stream?